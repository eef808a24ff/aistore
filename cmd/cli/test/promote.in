ais create bucket $BUCKET // IGNORE
mkdir -p /tmp/prm && echo 012345 > /tmp/prm/obj
ais promote /tmp/prm/obj ais://$BUCKET/one/one --keep=true // IGNORE
ais promote /tmp/prm/ ais://$BUCKET/three --keep=true // IGNORE
ais promote /tmp/prm ais://$BUCKET/fourth --keep=true // IGNORE
ais promote /tmp/prm ais://$BUCKET/fifth/ --keep=true // IGNORE
ais promote /tmp/prm/ ais://$BUCKET/sixth/ --keep=true // IGNORE

ais ls ais://$BUCKET

rm -rf /tmp/prm // IGNORE
ais rm bucket $BUCKET // IGNORE
