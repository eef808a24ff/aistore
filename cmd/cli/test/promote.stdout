NAME             SIZE
fifth/obj        7B
fourth/obj       7B
one/one          7B
sixth/obj        7B
three/obj        7B
